### ISSUE TITLE
Add current community observation, if it exists

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/react-nws#17]
> If a community has current obs, which Unalakleet does, display that as well. Kotlik does not have NWS current obs

Need a bit of help -- what exactly is the current community observation?  Is it literally just current conditions from an affiliated weather station?  How do we know what that is / where to access?

**Comment by takurkowski** [comment]
yes, it's just the current weather conditions. It may be in the same data feed as the forecasts, if not, we'll ask Rick!

Forecasts are given for the complete grid, but current conditions are only given where weather stations exist.
